NOTES.UNIX

```diff
@@ -89,3 +93,25 @@
 
     $ ./config --prefix=/usr/local/ssl --openssldir=/usr/local/ssl \
         '-Wl,--enable-new-dtags,-rpath,$(LIBRPATH)'
+
+ If you chose to use --run-path option, just pass additional
+ -Wl,--enable-new-dtags.
+
+ It might be worth noting that some/most ELF systems implement support
+ for relative runtime search path, relative to directory containing
+ the executable object, by interpreting $ORIGIN along with some other
+ internal variables. Consult your system documentation.
+
+ Linking your application
+ ------------------------
+
+ Above is the story of building and installing OpenSSL shared libraries
+ and application. But third-party applications dynamically linked with
```

Comment:
You can remove the whole first sentence and simply start with "Third party applications dynamically linked with ..."